ts/Core/Axis/OrdinalAxis.ts

```diff
@@ -984,15 +983,17 @@ namespace OrdinalAxis {
                         }
                     }
 
-                    if (series.isSeriesBoosting) {
-                        hasBoostedSeries = true;
-                    }
+                    // Enable boost for the ordinal axis, #14055.
+                    // if (series.isSeriesBoosting) {
+                    //     hasBoostedSeries = true;
+                    // }
```

Comment:
I think there's no need to keep commented out code. If this will cause any kind of regression in the future, we will simply bisect. The same applies to the code below